.github/workflows/blazorui.demo.cd.yml

```
@@ -113,6 +113,17 @@ jobs:
       env:
         ApiServerAddress: ${{ env.SERVER_ADDRESS }}
         WindowsUpdateSettings.FilesUrl: https://windows-components.bitplatform.dev
+
+    - name: Delete AssemblyOriginatorKeyFile.snk
+      run: |
+        del src\AssemblyOriginatorKeyFile.snk
+
+    - name: Extract strong sign certificate from env
+      uses: timheuer/base64-to-file@v1.2
+      with:
+          fileDir: '.\src\'
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)